### PR TITLE
BDOG-2048 - Return same response status

### PR DESCRIPTION
**Pull request by hmrc-phil**

### CODE REVIEW

**Review by colin-lamed — DISMISSED** [review]
Would it be simpler to have a single `WebhookController`, and just point to different urls depending on the path-param? Otherwise we have duplication of implementation and tests, and wouldn't scale if we require another one?

**Comment by hmrc-phil** [comment]
This is why I merged the connectors but yeah, I don't see why we shouldn't go one further.

File: conf/webhook.routes [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # microservice specific routes
 
-POST        /leak-detection             uk.gov.hmrc.platopsapi.leakdetection.LeakDetectionController.webhook()
-POST        /pr-commenter               uk.gov.hmrc.platopsapi.prcommenter.PrCommenterController.webhook()
\ No newline at end of file
+POST        /leak-detection             uk.gov.hmrc.platopsapi.webhook.WebhookController.leakDetection()
+POST        /pr-commenter               uk.gov.hmrc.platopsapi.webhook.WebhookController.prCommenter()
```
Comment (colin-lamed) [inline conf/webhook.routes:4]: you could have a single `Action` if you mapped `/leak-detection uk.gov.hmrc.platopsapi.webhook.WebhookController.webhook("leak-detection")`

**Review by colin-lamed — APPROVED** [review]

File: conf/webhook.routes (hunk 1 above)
Comment (hmrc-phil) [reply conf/webhook.routes:4]: I think I prefer it being explicit - Having a single action means I'd have to handle the string not matching

**Pull request merged**